### PR TITLE
Add S P A C E listing

### PR DESCRIPTION
**Pull request by stuartmscott**
This PR adds the S P A C E client with Fyne GUI to the list of Fyne applications.

### CODE REVIEW

File: _apps/space.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+screenshot3: https://space.aletheiaware.com/space-phone.png
+screenshot4: https://space.aletheiaware.com/space-table.png
+
+date:      2021-01-28 12:00:00
+excerpt:   Secure, Private, Storage.
+category:  utility
+developer: Aletheia Ware LLC
+
+git: https://github.com/AletheiaWareLLC/spacefynego.git
+package: aletheiaware.com/spacefynego
```
Comment (andydotxyz) [inline _apps/space.md:17]: I don't think this is correct, as I could not `fyne get` it, should it have "cmd" on the end?

```suggestion
package: aletheiaware.com/spacefynego/cmd
```

**Review by andydotxyz — COMMENTED** [review]
Cool :)
It didn't work for me unfortunately, but the suggested change should fix it.

**Review by andydotxyz — CHANGES_REQUESTED** [review]
Cool :)
It didn't work for me unfortunately, but the suggested change should fix it.

File: _apps/space.md (hunk 1 above)
Comment (stuartmscott) [reply _apps/space.md]: Ah quite right, thanks!

**Comment by stuartmscott** [comment]
Is there a way to define the tags that should be used? Specifically it needs `release` so that users are on the main network instead of the test (default), so it should be built with either `fyne release` or `go build -tag release`

**Comment by andydotxyz** [comment]
The release tag is used by the app listing app and `fyne get`

**Review by andydotxyz — APPROVED** [review]

**Comment by stuartmscott** [comment]
Perfect, thanks!

Can you merge - I don't have write access

**Comment by andydotxyz** [comment]
I was meaning to check you were happy before I did. It's in now :)

**Comment by stuartmscott** [comment]
The listing (https://apps.fyne.io/apps/space.html) uses "Space" instead of "S P A C E". Did I do something wrong, should `name:` instead be `title:`?

**Comment by stuartmscott** [comment]
Found a few issues;

1. The app gets installed as `cmd`, I will move everything from `cmd/` into a subdirectory `cmd/space` so the binary name is correct
2. Running `$ fyne get aletheiaware.com/spacefynego/cmd` resulted in;
```
Failed to set up installer: Missing application icon at "/Users/stuartscott/Documents/Projects/Go/src/aletheiaware.com/spacefynego/cmd/Icon.png"
```

**Comment by andydotxyz** [comment]
1. Thanks, if you can update we can push a revised listing
2. This is handled by the app installer and will be resolved by the future metadata file.
For now you can fix this by placing `Icon.png` at the location requested.

**Comment by andydotxyz** [comment]
> The listing (https://apps.fyne.io/apps/space.html) uses "Space" instead of "S P A C E". Did I do something wrong, should `name:` instead be `title:`?

Yes it should have been title. Was name used in one of the other files?

> Only screenshot1 is displayed, are the others used?

Not yet.

**Comment by stuartmscott** [comment]
Thanks Andy, these fixes are in #17.

> Yes it should have been title. Was name used in one of the other files?

I copied a previous PR (#5), must have been renamed since then

**Pull request merged**